src/components/shared/Button/Button.css

```diff
@@ -80,6 +80,12 @@
   background-color: disabledBackground;
 }
 
+/* Focused-visible for key tabbing */
+
+.main:focus-visible {
+  border: 1px var(--colony-black) solid;
```

Comment:
This ensures that there is some visual feedback when tabbing through inputs. Any suggestions on nicer alternative welcome, but I think some feedback is desirable.